aeron-client/src/main/c/command/aeron_control_protocol.h

```diff
@@ -208,6 +210,33 @@ typedef struct aeron_terminate_driver_command_stct
 }
 aeron_terminate_driver_command_t;
 
+typedef struct aeron_reject_image_command_stct
+{
+    aeron_correlated_command_t correlated;
+    int64_t image_correlation_id;
+    int64_t position;
+    int32_t reason_length;
+    uint8_t reason_text[1];
+}
+aeron_reject_image_command_t;
+
+struct aeron_publication_error_stct
+{
+    int64_t registration_id;
+    int64_t destination_registration_id;
+    int32_t session_id;
+    int32_t stream_id;
+    int64_t receiver_id;
+    int64_t group_tag;
+    int16_t address_type;
+    uint16_t address_port;
+    uint8_t address[16];
+    int32_t error_code;
+    int32_t error_message_length;
+    uint8_t error_message[1];
+};
```

Comment:
I did consider that, but I think it will be okay as all of the fields are currently on their matching alignment.  E.g. we have 64 bits between the `destination_registration_id` and the `receeiver_id`.  It should remain okay if it changes as long as 64 bit values are placed between `group_tag` and `address_type`.